### ISSUE TITLE
45410806 - Burnout Paradise

### ISSUE THREAD

**Issue by butterskeleton** [xenia-project/game-compatibility#1900]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-project/xenia/wiki/FAQ).
- [X] The Xenia build used is from the master branch. (not MLBS/AlexVS/Canary/pull requests, etc)
- [X] This issue isn't for tech support. (help with Xenia)
- [X] I've checked if this game doesn't already have a compatibility report.

### Xbox 360 Marketplace link

https://marketplace.xbox.com/en-US/Product/Burnout-Paradise/66acd000-77fe-1000-9115-d80245410806

### Xenia version

https://github.com/xenia-project/xenia/commit/c9073e101f38afe7b1444bae97eb8b78dcb151f9/

### Issues

the game boots up just fine, but as you get onto  the road rage next to the starter zone, after a few minutes the game crashes.

### Log

[log.zip](https://github.com/xenia-project/game-compatibility/files/7875477/log.zip)

### Screenshots

![Screenshot (7)](https://user-images.githubusercontent.com/65398582/149632295-bea12ca6-1ffa-496c-8147-3413027fe249.png)

### Labels

state-gameplay, apu-garbage

**Comment by Margen67** [comment]
Duplicate of #567 (?)

**Comment by butterskeleton** [comment]
No, this was the base game that came out before the ultimate box

**Comment by brad86m** [comment]
Regression within recent months. The game was working perfectly for a good while, but now crashes on loading before entering gameplay.

**Comment by Koriebonx98** [comment]
slight bug with starting up, few crashes. had to delete game save twice to work but after that boots in game fine. i think the title updates work however dlc don't.   tried with big surf island & cops and robbers

**Comment by Fanaticalight** [comment]
Tested on https://github.com/xenia-project/xenia/commit/3d30b2eec3ab1f83140b09745bee881fb5d5dde2/

Game is no longer regressed, tested on base 1.0 disc, playable, tested on D3D12 RTV. APU is still garbage but it gets more tolerable and less glitchy the more gameplay continues.

**Comment by brad86m** [comment]
> Tested on https://github.com/xenia-project/xenia/commit/3d30b2eec3ab1f83140b09745bee881fb5d5dde2/
> 
> Game is no longer regressed, tested on base 1.0 disc, playable, tested on D3D12 RTV. APU is still garbage but it gets more tolerable and less glitchy the more gameplay continues.

Well, it still regressed at you can't play the game on any builds from the past year-or-so. It crashes on boot, with absolutely no workaround but to use those old builds.

**Comment by brad86m** [comment]
Incorrect. The regression is still in place. It doesn't boot past the post-loading intro, AT ALL. It's been this way on all builds for the past year-or-so. There is no fix, yet. This goes for both normal and Canary builds.

**Comment by Sowa95** [comment]
Tested on: https://github.com/xenia-project/xenia/commit/3d30b2eec3ab1f83140b09745bee881fb5d5dde2

[xenia.zip](https://github.com/user-attachments/files/17463375/xenia.zip)

@brad86m when it comes to Master it's something on your end. It goes to gameplay consistently, default config.
On Canary it goes to gameplay with TU5, I don't know if that counts.

![1](https://github.com/user-attachments/assets/420be1e4-21ce-40c0-967c-944a1883005c)

<details><summary>Screenshot from Canary</summary>
<p>

![c](https://github.com/user-attachments/assets/0d0d2c4f-54e6-4618-a370-f47beadb03c8)

</p>
</details>

**Comment by brad86m** [comment]
My apologies for spreading misinformation. Not my intention. I'll try and see what the issue is on my end.

Nice to see it working in Canary, but I'm not sure what 'TU5' is ?

**Comment by Sowa95** [comment]
But TU5 doesn't bring base version to Ultimate Box version. What could be so wrong about 1mb patch file?

**Comment by brad86m** [comment]
> But TU5 doesn't bring base version to Ultimate Box version. What could be so wrong about 1mb patch file?

They had already changed default speed of cars (It's Burnout, they should be fast), damage toned down, and easier, AI opponents.

**Comment by Sowa95** [comment]
@brad86m Good to know, I didn't realize they rebalanced the gameplay so hard. If you want, we can clear the page a bit. I already deleted my message, and you can merge yours with the info about game versions.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/370

**Comment by RexhibHUB** [comment]
The game crashes on the loading screen using the latest version of canary
This is I would assume that it will only be because of my GPU unless using old versions of canary the game runs without very severe problems 

![Image](https://github.com/user-attachments/assets/f3a1cb0d-dd16-41a0-b800-4d4d34c42235)
![Image](https://github.com/user-attachments/assets/2c151959-76fc-43d1-946f-d329aff0ade7)

State-title
